### ISSUE TITLE
Deviation from Results in Paper

### ISSUE THREAD

**Issue by zw2700** [social-ai-uoft/ad-paper#1]
I have been trying to replicate the MinAtar results in the paper, but it seems my results do not match exactly with the ones reported in the paper. While results for Breakout and SpaceInvader are higher than the ones reported, results for Seaquest and Asterix are way lower than the ones reported. Please let me know what could be wrong! I've attached the hyperparameters I'm using below, which follows the description in the paper.

Replicated Average returns on MinAtar:
Breakout -~ 65
Freeway -~ 57
SpaceInvaer -~ 375
Seaquest -~ 1.5
Asterix -~ 6

`model_kwargs = dict(
        # Network architecture configuration
        net_arch=[400, 200, 200],
        input_skip_connections=True,
        recurrent_connections=True,
        forward_connections=True,
        average_predictions=True,  # Final prediction is the average of all layers
        layer_cls=ad.AttentionADCell,  # Use the attention layer
        # Inference configuration
        context_size=10,
        context_accumulation_alpha=1.0,  # Hard update, no accumulation
        # Training configuration
        learning_rate=1e-4,
        huber_loss=False,  # Use standard MSE loss
        buffer_size=5_000_000,
        target_network_frequency=1000,
        max_grad_norm=1.0,
        batch_size=512,
        start_eps=1.0,
        end_eps=0.01,
        exploration_fraction=0.1,
        learning_starts=50_000,
        train_frequency=4,
        seed=args.seed,
        double_q=True,
        gamma=0.99
    )`
